server/database/queries/members/addMember.js

```diff
@@ -0,0 +1,12 @@
+const dbconnection = require('../../config/db_connection');
+
+const addMmeber = (memberInfo) => {
+  const queryValues = Object.values(memberInfo);
+  const sql = {
+    text: 'INSERT INTO member(username,email, pass, avatar) VALUES ( $1, $2, $3, $4 ) RETURNING *',
+    values: [...queryValues, 'https://png.pngtree.com/svg/20161027/631929649c.svg'],
```

Comment:
@Fatmasiam I know this hard-coded avatar url might just be temporary but here's my suggestion for handling "default" avatar pictures:  Instead of storing a default url in the database for a user who doesn't have an avatar picture, leave the database field empty and add a default url in the frontend instead - this lets you change the default image (if your frontend design changes) with a simple frontend code change instead of having to update many database rows.

Reply:
@tomduggan85, thanx a lot, that is better for performance